### DISCUSSION TITLE
refresh overall price for a part needs to be done manually

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7963] [Q&A]
Hi,
   I just realize that if BOM is added and I don't click the refresh button on the pricing, it just wouldn't update. I'm wondering if there is any api that can do this automatically? 
Best regards.

**Comment by SchrodingersGat** [comment]
See https://github.com/inventree/InvenTree/issues/7955

**Reply by jesseyu1984918** [reply]
I think the issue that guy described is slightly difference. It occurs to me that the price in the BOM list is updated properly, however, the part's overall pricing is not updated automatically and thus breaks the linke if THE part is used as a BOM for another part.

**Reply by jesseyu1984918** [reply]
Also that if I have override or refresh the price, when export the BOM, it would double count the part and the BOM...This is a bit confusing as for my use case usually the part price would be the sum of BOM, but there might not be a good solution for all.
